packages/devtools_app/lib/src/network/network_controller.dart

```diff
@@ -20,8 +20,17 @@ class NetworkController {
   }
 
   /// Notifies that new HTTP requests have been processed.
-  ValueListenable<HttpRequests> get requestsNotifier => _httpRequestsNotifier;
-  final _httpRequestsNotifier = ValueNotifier<HttpRequests>(HttpRequests());
+  ValueListenable<HttpRequests> get requests => _requests;
+  final _requests = ValueNotifier<HttpRequests>(HttpRequests());
+
+  void selectHttpRequest(HttpRequestData selection) {
+    _selectedHttpRequest.value = selection;
+  }
```

Comment:
move functions below variables and getters declared in this file

Reply:
will do